src/main/thrift/parquet.thrift

```diff
@@ -211,6 +206,97 @@ struct Statistics {
    4: optional i64 distinct_count;
 }
 
+/** Empty structs to use as logical type annotations */
+struct StringType {}
+struct MapType {}
+struct ListType {}
+struct EnumType {}
+struct DateType {}
+
+/**
+ * Logical type to annotate a column that is always null.
+ *
+ * Sometimes when discovering the schema of existing data values are always
+ * null and the physical type is assumed.
+ */
+struct NullType {}
+
+/**
+ * Decimal logical type annotation
+ *
+ * To maintain forward-compatibility in v1, implementations using this logical
+ * type must also set scale and precision on the annotated SchemaElement.
+ */
+struct DecimalType {
+  1: required i32 scale
+  2: required i32 precision
+}
+
+/** Time units for logical types */
+struct MilliSeconds {}
+struct MicroSeconds {}
+union TimeUnit {
+  1: MilliSeconds MILLIS
+  2: MicroSeconds MICROS
+}
+
+/** Timestamp logical type annotation */
+struct TimestampType {
+  1: required bool isAdjustedToUTC
```

Comment:
The model we discussed for Parquet is different from the Arrow spec.
 In Arrow, both `Timestamp` and `TimestampTz` are stored in UTC. The `writer Timezone` string is additionally stored to re-compute the `Timestamp` values.
For Parquet, the idea is to store `Timestamp` values **as is** from epoch without conversion to UTC.
The conversion to UTC happens only for `TimestampTz` values.
The parquet approach is slightly efficient since we don't have to re-compute the `Timestamp` values thereby enabling bulkload of the column.
 The presence of a `writer Timezone` in the file metadata will also prohibit concatenation of files from different `Timezones`.